### PR TITLE
fix: fix webhook repo api

### PR DESCRIPTION
**Pull request by abhinav-harness**
## Comment Triggers

<details>
  <summary>PR Check triggers</summary>
  
- Build: `trigger build`
- gitleaks: `trigger gitleaks`

### CODE REVIEW

File: harness/code/docs/RepoSoftDeleteResponse.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# RepoSoftDeleteResponse
+
+## Properties
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**DeletedAt** | **int32** |  | [optional] [default to null]
```
Comment (atefehmohseni) [inline harness/code/docs/RepoSoftDeleteResponse.md:6]: out of curiosity, epochs are int64 in Gitness, is there a reason it's int32 here?

**Review by atefehmohseni — CHANGES_REQUESTED** [review]

File: harness/code/api_status_checks.go [hunk 2]
```diff
@@ -11,12 +11,11 @@ package code
 import (
 	"context"
 	"fmt"
+	"github.com/antihax/optional"
```
Comment (johannesHarness) [inline harness/code/api_status_checks.go:14]: usually non-native libraries are coming later in import sortings?

**Review by johannesHarness — APPROVED** [review]

File: harness/code/docs/RepoSoftDeleteResponse.md (hunk 1 above)
Comment (abhinav-harness) [reply harness/code/docs/RepoSoftDeleteResponse.md:6]: https://harness.atlassian.net/browse/CODE-1586

File: harness/code/api_pullreq.go [hunk 3]
```diff
@@ -26,6 +25,163 @@ var (
...
+		}
+	}
+
+	if localVarHttpResponse.StatusCode >= 300 {
+		newErr := GenericSwaggerError{
+			body:  localVarBody,
+			error: localVarHttpResponse.Status,
+		}
+		if localVarHttpResponse.StatusCode == 200 {
+			var v []TypesPullReqChecks
```
[the remark this answers is not in the document]
Comment (abhinav-harness) [reply harness/code/api_pullreq.go:130]: auto generated :P

File: harness/code/docs/RepositoryApi.md [hunk 4]
```diff
@@ -1020,6 +1022,45 @@ Name | Type | Description  | Notes
 
 [[Back to top]](#) [[Back to API list]](../README.md#documentation-for-api-endpoints) [[Back to Model list]](../README.md#documentation-for-models) [[Back to README]](../README.md)
 
+# **PurgeRepository**
+> PurgeRepository(ctx, accountIdentifier, deletedAt, repoIdentifier, optional)
```
[the remark this answers is not in the document]
Comment (abhinav-harness) [reply harness/code/docs/RepositoryApi.md:1026]: did you update it later?

[the remark this answers is not in the document]
Comment (abhinav-harness) [reply harness/code/docs/RepositoryApi.md:1026]: ohh its autogenerated and upto client, not in my hand

**Review by atefehmohseni — APPROVED** [review]

**Pull request merged**